lib/stimulus_reflex/channel.rb

```diff
@@ -22,11 +22,11 @@ def receive(data)
     reflex_name, method_name = target.split("#")
     reflex_name = reflex_name.classify
     arguments = data["args"] || []
-    options = StimulusReflex::DomElement.new(data["attrs"])
+    element = StimulusReflex::Element.new(data["attrs"])
```

Comment:
Been thinking about this more and I still like element.

Seems inline with the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset)